compiler/testSuite/08-error/check7-v.bal

```diff
@@ -0,0 +1,7 @@
+public function main() {
+    checkpanic newError();
+}
+
+function newError() returns error {
+    return error("failed successfully");
+}
```

Comment:
This is PR is not done yet, I need to fix case.
From the spec, this is a valid case, even though it's failing in jBallerina beta2.

Reply:
I plan to fix this by passing a boolean to `codeGenCheckingExpr`